src/NetscriptFunctions/Formulas.ts

```diff
@@ -134,6 +135,12 @@ export function NetscriptFormulas(): InternalAPI<IFormulas> {
         checkFormulasAccess(ctx);
         return donationForRep(reputation, person);
       },
+      sharePower: (ctx) => (_threads, _cpuCores) => {
```

Comment:
Most things that take cores as an argument have it as an optional argument, and it seems like that would be a good fit here as well.